target/linux/ramips/image/rt305x.mk

```diff
@@ -1192,6 +1192,14 @@ define Device/zyxel_keenetic
 endef
 TARGET_DEVICES += zyxel_keenetic
 
+define Device/zyxel_keenetic-lite_rev.b
+  SOC := rt5350
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := Keenetic Lite rev.B
```

Comment:
I'd go for
DEVICE_MODEL := Keenetic Lite
DEVICE_VARIANT := B

Reply:
fixed